nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -404,4 +161,4 @@ def is_docbook(o, key):
         file=sys.stderr)
     sys.exit(1)
 
-json.dump(convertMD(unpivot(options)), fp=sys.stdout)
```

Comment:
@pennae, this is a breaking change for the commonmark and asciidoc generators. Now that all of the markdown machinery is in the docbook generator, could we merge all three generators into one?